server/mscalendar/mscalendar.go

```diff
@@ -18,6 +18,7 @@ type MSCalendar interface {
 	EventResponder
 	Subscriptions
 	Users
+	Welcome(userID string) error
```

Comment:
(1/5) MSCalendar is not a welcomer, the bot in the dependencies is. But MSCalendar must know how to greet a new user, to be used on the commands, so that is why we expose that function on the interface.

Reply:
Dunno, I'd advocate composing the entire interface in. What's the harm? Polluting MSCalendar namespace?